lua/midi.lua

```diff
@@ -138,6 +138,19 @@ function Midi.connect(n)
   d.channel_pressure = function(val, ch)
       d.send{type="channel_pressure", val=val, ch=ch or 1}
     end
+  d.start = function()
+      d.send{0xfa}
```

Comment:
readability (less magic numbers) would be

```
d.send{type="start"}
```

since you already created a `type` for `start`

etc for others...

Reply:
oh - nice. I'll update

Reply:
Hmm... in that case wouldn't the magic numbers need to be included in `Msg.to_data` instead?